docs: update angular integration for >9 versions (#1000)
elastic/apm-agent-rum-js: docs/angular-integration.asciidoc

[source,bash]
----
npm install @elastic/apm-rum-angular@alpha --save
----

> NOTE: If you are in Angular version < 9.0, please use @elastic/apm-rum-angular 1.x versions for
> instrumenting your application.

Reviewer: Seeing as Angular <9 is EOL, I don't think it makes sense to maintain a separate section for the old docs. However, the old docs will continue to live on in version 4.x of the RUM agent reference, so we can link there as a convenience for now.

```suggestion
NOTE: If you are using an Angular version < 9.0, use @elastic/apm-rum-angular 1.x to instrument your application. Details are available in a https://www.elastic.co/guide/en/apm/agent/rum-js/4.x/angular-integration.html[prior release].
```

Reviewer: If a new major is releasing for this change, we can edit the link above to point to version 5.x instead of 4.x

Author: Not really the major is only going to be effective for Angular package not for other packages, so we can reference to 4.x versions. Good idea 👍
